test/framework/vsphere.go

```diff
@@ -167,6 +172,15 @@ func WithBottleRocket121() VSphereOpt {
 	}
 }
 
+func WithPrivateNetwork() VSphereOpt {
+	return func(v *VSphere) {
+		v.fillers = append(v.fillers,
+			api.WithStringFromEnvVar(vspherePrivateNetworkVar, api.WithNetwork),
+		)
+		v.cidr = os.Getenv(privateNetworkCidrVar)
```

Comment:
Also validate here?

Reply:
This is validated inside the `NewVSphere`